src/main/java/net/glease/ggfab/mte/MTE_AdvAssLine.java

```diff
@@ -129,7 +142,8 @@ public class MTE_AdvAssLine extends GT_MetaTileEntity_ExtendedPowerMultiBlockBas
     private long baseEUt;
     private boolean stuck;
 
-    private final ArrayList<GT_MetaTileEntity_Hatch_DataAccess> mDataAccessHatches = new ArrayList<>();
+    private final List<GT_MetaTileEntity_Hatch_DataAccess> mDataAccessHatches = new ArrayList<>();
+    private final ItemStack[] mItemCache = new ItemStack[16];
```

Comment:
could just call this itemCache and drop the m imo.

Reply:
doesn't hurt to keep it.